README.md

```diff
@@ -235,6 +252,55 @@ like this:
 You'll probably get a more reliable experience if you use
 [forever](https://npmjs.org/package/forever) to keep the daemon running.
 
+
+### Environment
+
+Set the env variable `NODE_DEBUG` to enable debugging.
+
+Example:
+
+    export NODE_DEBUG=dev,all,net,http,fs,tls,module,timers
+
+See [How to set NODE_DEBUG](http://www.juliengilli.com/2013/05/26/Using-Node.js-NODE_DEBUG-for-fun-and-profit/)
+
+
+## Using the command line tools
+
+### pump-register-app
+
+First use this tool to create the credentials file
+
+    ./bin/pump-register-app  -t <APPNAME>
+
+This will create the file `~/.pump.d/*server*.json` that contains your credentials.
```

Comment:
register app is app name not server.
```
.describe("t", "Title of the app")
```
https://github.com/e14n/pump.io/blob/1029ac71b94dec2b51f6b2aa461b5b2a514e585a/bin/pump-register-app#L39